### PR TITLE
Gr ch37 trunk

### PR DESCRIPTION
**Pull request by B2Pi**
Merge trunk back in...

### CODE REVIEW

File: src/scripts/prepareCiceroInput.pl [hunk 1]
```diff
@@ -33,46 +33,63 @@
...
+  my($fname, $fdesc) = @_;
+  if (!defined($fname) || $fname eq '') {
+    die "No $fdesc has been defined";
+  } elsif (!-f $fname) {
+    die "$fdesc file $fname not found"
+  }
 }
 
 my $black_list_file = $conf->{'BLACKLIST_GENES'};
-open(my $BLK, "<", "$black_list_file");
```
Comment (adthrasher) [inline src/scripts/prepareCiceroInput.pl:50]: This uses older names for the reference files.

File: src/scripts/prepareCiceroInput.pl [hunk 2]
```diff
@@ -33,46 +33,63 @@
...
+validate_filename($black_list_file, 'blacklist');
```
Comment (adthrasher) [inline src/scripts/prepareCiceroInput.pl:62]: I don't think we need this. If you rebase, all of these `open` commands should have a `|| die ` check to ensure that the file exists.

File: src/scripts/get_cicero_cmds.pl [hunk 3]
```diff
@@ -9,21 +9,22 @@
...
-my $queue = $ENV{"AFC_DEFAULT_QUEUE"}; 
+my $queue = $ENV{"AFC_DEFAULT_QUEUE"};
 
 my ( $help, $man, $version, $usage );
 my $bam_file;
-my $genome; 
+my $genome;
 my $read_length = 100;
 my $output_dir;
+my $cluster_arg = 10;
```
Comment (adthrasher) [inline src/scripts/get_cicero_cmds.pl:27]: This is in master already, please rebase the branch to pull in those changes that will make reviewing this much more straightforward.

**Review by adthrasher — APPROVED** [review]

**Review by B2Pi — COMMENTED** [review]
Looks good to me. Sorry for the white space changes...

**Comment by adthrasher** [comment]
> Looks good to me. Sorry for the white space changes...

No worries, I prefer the changes you've made. I just haven't systematically gone through to remove the excessive white space.

**Pull request merged**